UPGRADING.md

```diff
@@ -1,6 +1,39 @@
 Upgrading Grape
 ===============
 
+### Upgrading to >= 1.2.5
+
+[Virtus](https://github.com/solnic/virtus) isn't used for coercing anymore. If your project depends on Virtus, you have to add it to your `Gemfile`. Also, if Virtus is used as a custom type
+
+```ruby
+class User
+  include Virtus.model
+
+  attribute :id, Integer
+  attribute :name, String
+end
+
+# somewhere in your API
+params do
+  requires :user, type: User
+end
+```
+
+you need to add a class-level `parse` method to the model:
+
+```ruby
+class User
+  include Virtus.model
+
+  attribute :id, Integer
+  attribute :name, String
+
+  def self.parse(attrs)
+    new(attrs)
+  end
+end
+```
+
```

Comment:
I think we want to explain how to rewrite a custom type with dry-types here, first. If I am upgrading I'd rather not use Virtus unless I can't.

Reply:
Dry-types cannot be used in place of Virtus for custom types, [dry-schema](https://dry-rb.org/gems/dry-schema/1.4/) might be. So, if users need something like Virtus they need to keep  it or migrate to dry-schema. I guess it is up to them to decide, the goal is to avoid breaking changes, so if they use Virtus, they can still use it until they have time to migrate.

Reply:
It sounds like you're saying that if you want a custom type you *have* to use Virtus? Or are you saying you *can* use Virtus? Either way let's make this very clear in the upgrading/readme doc.

Reply:
No, Virtus isn't required for custom types. As it was before, any class implementing the `parse` method might be used. :slightly_smiling_face: I've added an additional sentence to the upgrading doc.